apps/src/blockly/addons/cdoXml.js

```diff
@@ -47,7 +78,11 @@ export default function initializeBlocklyXml(blocklyWrapper) {
     // flowing from top to bottom. Blocks with absolute Y positions
     // do not influence the placement of other blocks.
     let cursor = {
-      x: blockSpace.RTL ? width - padding : padding,
+      x: blockSpace.RTL
```

Comment:
Seems like there's a change to the positioning rules here, could we update the comment to reflect that change?

Reply:
Updated and moved up a few lines.